FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -1060,7 +1063,7 @@ extension ComposeViewController {
             title: "compose_password_modal_title".localized,
             message: "compose_password_modal_message".localized,
             preferredStyle: .alert
-        )
+        ).popoverPresentation(style: .centred(view))
```

Comment:
`popoverPresentation` not needed here too

Reply:
You will get a crash in iPad it popoverPresentationStyle is not set